src/analysis/processing/qgsalgorithmexportmesh.cpp

```diff
@@ -145,29 +145,29 @@ void QgsExportMeshOnElement::initAlgorithm( const QVariantMap &configuration )
 {
   Q_UNUSED( configuration );
 
-  addParameter( new QgsProcessingParameterMeshLayer( QStringLiteral( "INPUT" ), QObject::tr( "Input Mesh Layer" ) ) );
+  addParameter( new QgsProcessingParameterMeshLayer( QStringLiteral( "INPUT" ), QObject::tr( "Input mesh layer" ) ) );
 
 
   addParameter( new QgsProcessingParameterMeshDatasetGroups(
                   QStringLiteral( "DATASET_GROUPS" ),
-                  QObject::tr( "Dataset Groups" ),
+                  QObject::tr( "Dataset groups" ),
                   QStringLiteral( "INPUT" ),
                   supportedDataType(), true ) );
 
   addParameter( new QgsProcessingParameterMeshDatasetTime(
                   QStringLiteral( "DATASET_TIME" ),
-                  QObject::tr( "Dataset Time" ),
+                  QObject::tr( "Dataset time" ),
                   QStringLiteral( "INPUT" ),
                   QStringLiteral( "DATASET_GROUPS" ) ) );
 
-  addParameter( new QgsProcessingParameterCrs( QStringLiteral( "CRS_OUTPUT" ), QObject::tr( "Output Coordinate System" ), QVariant(), true ) );
+  addParameter( new QgsProcessingParameterCrs( QStringLiteral( "CRS_OUTPUT" ), QObject::tr( "Output coordinate system" ), QVariant(), true ) );
 
   QStringList exportVectorOptions;
   exportVectorOptions << QObject::tr( "Cartesian (x,y)" )
                       << QObject::tr( "Polar (magnitude,degree)" )
-                      << QObject::tr( "Cartesian and Polar" );
-  addParameter( new QgsProcessingParameterEnum( QStringLiteral( "VECTOR_OPTION" ), QObject::tr( "Export Vector Option" ), exportVectorOptions, false, 0 ) );
-  addParameter( new QgsProcessingParameterFeatureSink( QStringLiteral( "OUTPUT" ), QObject::tr( "Output Vector Layer" ), sinkType() ) );
+                      << QObject::tr( "Cartesian and polar" );
```

Comment:
Actually this one was correct before -- enum values are title case :laughing: 

We follow the HIG guidelines here. I.e. labels (the parameter names) must be sentence case, while combo box items (the enum values) are title case. (This is inherited from Qt/KDE hig)

(and for reference -- action names = title case, dialog titles = title case, button text = title case, checkbox labels = sentence case, group box titles = title case (unless they are acting more like a check box and placed alongside other checkboxes, in which case they should be sentence case to match their surroundings)